HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/PaymentComponentsController+Helpers.swift

```diff
@@ -0,0 +1,806 @@
+     localized strings, then returns a `PaymentInfoViewController` with the view model.
+
+     - Returns: A configured `UIViewController` for displaying payment information.
+     */
+    func paymentInfoViewController() -> UIViewController {
+        let paymentInfoViewModel = PaymentInfoViewModel(paymentProviders: paymentProviders,
+                                                        configuration: configurationProvider.paymentInfoConfiguration,
+                                                        strings: stringsProvider.paymentInfoStrings,
+                                                        poweredByGiniConfiguration: configurationProvider.poweredByGiniConfiguration,
+                                                        poweredByGiniStrings: stringsProvider.poweredByGiniStrings)
+        return PaymentInfoViewController(viewModel: paymentInfoViewModel)
+    }
+
+    /**
+     Provides a custom Gini view for installing the app if not present.
+
+     This method initializes an `InstallAppBottomViewModel` with the necessary configurations and
+     localized strings, and returns an `InstallAppBottomView` configured with the view model.
+
+     - Returns: A configured `BottomSheetViewController` for the app installation process.
+     */
+    public func installAppBottomSheet() -> BottomSheetViewController {
+        previousPresentedView = nil
+        let installAppBottomViewModel = InstallAppBottomViewModel(selectedPaymentProvider: healthSelectedPaymentProvider,
+                                                                  installAppConfiguration: configurationProvider.installAppConfiguration,
+                                                                  strings: stringsProvider.installAppStrings,
+                                                                  primaryButtonConfiguration: configurationProvider.primaryButtonConfiguration,
+                                                                  poweredByGiniConfiguration: configurationProvider.poweredByGiniConfiguration,
+                                                                  poweredByGiniStrings: stringsProvider.poweredByGiniStrings)
+        installAppBottomViewModel.viewDelegate = self
+        let installAppBottomView = InstallAppBottomView(viewModel: installAppBottomViewModel, bottomSheetConfiguration: configurationProvider.bottomSheetConfiguration)
+        return installAppBottomView
+    }
+    
+    /**
+     Provides a custom Gini view for onboarding the user about the sharing invoices flow.
+
+     This method initializes a `ShareInvoiceBottomViewModel` with the necessary configurations and
+     localized strings, and returns a `ShareInvoiceBottomView` configured with the view model.
+     It also increments the onboarding count for the selected payment provider.
+
+     - Parameter documentId: An optional identifier for the document associated with the invoice.
```

Comment:
[clarification] Can we clean the documentation by removing unneeded parameters like documented?